components/features/galleryAssets/web/buy-action.tsx

```
@@ -0,0 +1,167 @@
+        } catch (e) {
+            console.error(e);
+            return null;
+        }
+    };
+
+    const handleBuyComplete = async () => {
+        if (!profile) return;
+        try {
+        } catch (e) {
+            console.error(e);
+        }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)